cmd/api/src/daemons/datapipe/models.go

```diff
@@ -195,16 +205,31 @@ type ConvertedData struct {
 	RelProps  []ein.IngestibleRelationship
 }
 
+func (s *ConvertedData) Clear() {
+	s.NodeProps = make([]ein.IngestibleNode, 0)
+	s.RelProps = make([]ein.IngestibleRelationship, 0)
```

Comment:
If the intent is for this data structure to be instantiated and cleared for re-use then we can optimize this to not reallocate space:

```golang
s.NodeProps = s.NodeProps[:0]
```